File: Visual Basic/PROJECTS/repos/ConverterFunction/ConverterFunction/Form1.vb
Public Class Form1

    Private Function Inches(ByVal Cm As String) As Double
        Inches = Cm / 2.54
        Inches = FormatNumber(Inches, 2)
        Return Inches
    End Function

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Label1.Text = Inches(TextBox1.Text) & " Inches"
    End Sub
End Class

File: Visual C#/SRC/Code/Message/Form1.cs
namespace Message
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            try
            {
                throw new NotImplementedException();
            }
            catch (NotImplementedException notImp)
            {
                DialogResult result =
                     MessageBox.Show("Proceed?", notImp.Message, MessageBoxButtons.YesNo, MessageBoxIcon.Error);

                textBox1.Text = result.ToString();
                if (result == DialogResult.Yes) label1.Text = "Proceeding...";
                if (result == DialogResult.No) label1.Text = "Stopping...";
            }
        }
    }
}